apps/labs/next.config.js

```diff
@@ -35,8 +35,10 @@ const nextConfig = {
         //   Content-Type: application/octet-stream
         //
         // The following rule catches some plain text file types that we link
-        // to, and supplies the text/plain MIME type in the Content-Type header.
-        source: '/:path*(.py|.cpp)',
+        // to, and supplies the text/plain MIME type in the Content-Type header
+        // so that end users can open and view the file directly in their
+        // browser rather than having to first download and save the file.
+        source: '/:path*.(py|cpp)',
```

Comment:
May want to add a comment here as to why this pattern works... that only the stuff inside the parentheses is interpreted as regex, and thus the period outside the parens is a period literal, not a regex wildcard.

Could also link to the [relevant section of Next docs](https://nextjs.org/docs/api-reference/next.config.js/headers#header-overriding-behavior) for good measure?